repos/tap/components/sidebar/content/fileTreePanel.js

```diff
@@ -59,13 +59,16 @@ const FileTreeHeader = ({styles, title}) => {
 export const FileTreePanel = (props) => {
   const theme = useTheme()
   const styles = theme.get('sidebar.content')
+  console.log(styles)
```

Comment:
You already mentioned you would remove this, so I'm just putting a comment here for so I remember to check it